pkl-core/src/main/java/org/pkl/core/runtime/VmMap.java

```diff
@@ -98,7 +99,7 @@ public boolean hasNext() {
       @TruffleBoundary
       public Map.Entry<Object, Object> next() {
         var key = keyIterator.next();
-        return Map.entry(key, map.get(key));
+        return Map.entry(key, Objects.requireNonNull(map.get(key)));
```

Comment:
Mmm I'm not sure why GitHub's diff is showing this, but this is out of date with what's on our main branch.

We already have a non-null assertion here, see:

https://github.com/apple/pkl/blob/a9c4963a4a60e06af683f4a700f290b75ca99d22/pkl-core/src/main/java/org/pkl/core/runtime/VmMap.java#L101-L103